examples/authResolvers.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs')
+const path = require('path')
+const express = require('express')
+const session = require('express-session')
+const Keycloak = require('keycloak-connect')
+const { ApolloServer, gql } = require('apollo-server-express')
+
+const { KeycloakContextProvider, KeycloakTypeDefs, auth, hasRole } = require('../')
+
+const app = express()
+
+const memoryStore = new session.MemoryStore()
+
+const graphqlPath = '/graphql'
+
+
+app.use(session({
+  secret: 'secret',
```

Comment:
potential blocker for merge

Reply:
Agree I can easily replace that with something more obvious